### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by lemairecarl** [vitalab/vitalab.github.io#23]
#### Classification / Feature extractors

- [x] LeNet (Gradient-Based Learning Applied to Document Recognition)  
http://yann.lecun.com/exdb/publis/pdf/lecun-01a.pdf
- [x] AlexNet (ImageNet Classification with Deep Convolutional Neural Networks)
http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf
- [x] GoogLeNet, Inception (Going Deeper with Convolutions)  
https://arxiv.org/abs/1409.4842
- [x] VGGNet
- [x] Inception v2/v3 (Rethinking the Inception Architecture for Computer Vision)
https://arxiv.org/abs/1512.00567
- [x] ResNet
- [ ] Identity Mappings in Deep Residual Networks
https://arxiv.org/abs/1603.05027
- [x] Xception: Deep Learning with Depthwise Separable Convolutions
https://arxiv.org/abs/1610.02357
- [x] DenseNet (Densely Connected Convolutional Networks)  
https://arxiv.org/abs/1608.06993
- [x] DelugeNets: Deep Networks with Massive and Flexible Cross-layer Information Inflows
https://arxiv.org/abs/1611.05552
- [ ] Deeply-Supervised Nets  
https://arxiv.org/abs/1409.5185
- [x] Striving for Simplicity: The All Convolutional Net (All-CNN)
https://arxiv.org/abs/1412.6806
- [x] SqueezeNet: AlexNet-level accuracy with 50x fewer parameters and <0.5MB model size
https://arxiv.org/abs/1602.07360

#### Detection, Localization, Segmentation

- [x] OverFeat: Integrated Recognition, Localization and Detection using Convolutional Networks  
https://arxiv.org/abs/1312.6229
- [x] FCNN: Fully Convolutional Networks for Semantic Segmentation
https://people.eecs.berkeley.edu/~jonlong/long_shelhamer_fcn.pdf
https://computing.ece.vt.edu/~f15ece6504/slides/L13_FCN.pdf
- [x] Siamese Net : Learning a Similarity Metric Discriminatively, with Application to Face Verification
http://yann.lecun.com/exdb/publis/pdf/chopra-05.pdf
- [x] ConvDeconv : Learning Deconvolution Network for Semantic Segmentation
http://www.cv-foundation.org/openaccess/content_iccv_2015/papers/Noh_Learning_Deconvolution_Network_ICCV_2015_paper.pdf
- [x] R-CNN (Rich feature hierarchies for accurate object detection and semantic segmentation)
https://arxiv.org/pdf/1311.2524.pdf
- [x] Fast R-CNN  
https://arxiv.org/abs/1504.08083
- [x] Faster R-CNN: Towards Real-Time Object Detection with Region Proposal Networks  
https://arxiv.org/abs/1506.01497
- [x]  The One Hundred Layers Tiramisu: Fully Convolutional DenseNets for Semantic Segmentation 
https://arxiv.org/pdf/1611.09326.pdf
- [x] YOLO
- [x] SSD
- [x] Feature Pyramid Network for Object Detection
https://arxiv.org/pdf/1612.03144.pdf
- [x] Network In Network --- 1x1 conv
https://arxiv.org/abs/1312.4400
- [x] DeepMask
https://arxiv.org/pdf/1506.06204.pdf
- [x] SharpMask
https://arxiv.org/pdf/1603.08695.pdf
- [x] The Importance of Skip Connections in Biomedical Image Segmentation

-----------------------------------------------
### Fundamental research / Network design

- [x] Neural Architecture Search with Reinforcement Learning
https://arxiv.org/abs/1611.01578
- [ ] Visualizing and Understanding Convolutional Networks
https://arxiv.org/abs/1311.2901
- [ ] Intriguing properties of neural networks (fooling images)  
https://arxiv.org/abs/1312.6199
- [ ] Explaining and Harnessing Adversarial Examples  
https://arxiv.org/abs/1412.6572
- [ ] Deep Inside Convolutional Networks: Visualising Image Classification Models and Saliency Maps  
https://arxiv.org/abs/1312.6034
- [ ] Understanding Neural Networks Through Deep Visualization
https://arxiv.org/abs/1506.06579
- [x] Learning both Weights and Connections for Efficient Neural Networks
https://arxiv.org/abs/1506.02626
- [ ] Deep Compression: Compressing Deep Neural Networks with Pruning, Trained Quantization and Huffman Coding
https://arxiv.org/abs/1510.00149
- [ ] DSD: Dense-Sparse-Dense Training for Deep Neural Networks
https://arxiv.org/abs/1607.04381
- [ ] Dropout as a Bayesian Approximation: Representing Model Uncertainty in Deep Learning  
https://arxiv.org/abs/1506.02142

-----------------------------------------------
### Semi-supervised Learning

- [ ] Semi-Supervised Learning with Ladder Networks
https://arxiv.org/abs/1507.02672
- [ ] Deconstructing the Ladder Network Architecture  
https://arxiv.org/abs/1511.06430

-----------------------------------------------
### Approximate Nearest Neighbor

- [ ] Scalable Nearest Neighbor Algorithms for High Dimensional Data
http://www.cs.ubc.ca/research/flann/uploads/FLANN/flann_pami2014.pdf
- [ ] Fast Matching of Binary Features
http://www.cs.ubc.ca/research/flann/uploads/FLANN/binary_matching_crv2012.pdf
- [ ] Fast Approximate Nearest Neighbors with Automatic Algorithm Configuration
http://www.cs.ubc.ca/research/flann/uploads/FLANN/flann_visapp09.pdf

-----------------------------------------------
### Other

- [ ] Practical Recommendations for Gradient-Based Training of Deep Architectures  
https://arxiv.org/pdf/1206.5533.pdf
- [x] A Few Useful Things to Know about Machine Learning
http://homes.cs.washington.edu/~pedrod/papers/cacm12.pdf
- [ ] Learning Multiple Layers of Features from Tiny Images
http://www.cs.toronto.edu/~kriz/learning-features-2009-TR.pdf
- [ ] t-SNE
http://lvdmaaten.github.io/tsne

-----------------------------------------------

### Lecture/course notes
- [x] Support Vector Machines (CS229 Lecture notes)
https://see.stanford.edu/materials/aimlcs229/cs229-notes3.pdf

**Comment by lemairecarl** [comment]
@jodoin Please edit the list above to remove stuff that you don't think should be on the site.

**Comment by jodoin** [comment]
They all disserve to be on the site.  They might not all be "essentials" however.

**Comment by Dref360** [comment]
I'll take Siamese since I already worked with those

**Comment by czotti** [comment]
I take ConvDeconv!
